limbo/plugins/oxford.py

```diff
@@ -0,0 +1,59 @@
+"""!define - Returns the oxford dictionary definition of the given term"""
+import os
+import re
+import requests
+
+def define(word):
+    app_id = os.environ.get('OXFORD_APP_ID', '')
```

Comment:
hah yeah it does, and it's also not a big deal. Dunno what I was thinking about when I wrote that.